### ISSUE TITLE
Persisting Neo4j data

### ISSUE THREAD

**Issue by snewhouse** [mhelvens/docker-node-neo4j-example#2]
Hi, have you tried something like this?

`--volume=$HOME/neo4j/data:/data` : will add a volume (/data) to allow the database to be persisted outside its container

``` bash
docker run \
--publish=7474:7474 \
--publish=80:80 \
--volume=$HOME/neo4j/data:/data \
mhelvens/docker-node-neo4j-example:latest
```

more soon

**Comment by mhelvens** [comment]
Yep. Sure. That's been the obvious way to go. Although as I understand it, Docker now supports in-container storage of some sort, so the user of our image doesn't have to keep track of a directory anywhere.
